xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -853,10 +901,11 @@ If the look up text is invalid or incomplete UTF-16 text (such as unpaired surro
         <Parameter Name="value" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">If the method succeeds, a byte array that contains the base 64-encoded JSON string.</param>
+        <summary>Tries to parse the current JSON token value from the source and decodes the base 64 encoded JSON string as a byte array.</summary>
+        <returns><see langword="true" /> if the entire token value is encoded as valid base 64 text and can be successfully decoded; <see langword="false" /> otherwise.</returns>
```

Comment:
To be consistent with the JsonElement API doc:
```suggestion
        <returns><see langword="true" /> if the entire token value is encoded as valid base 64 text and can be successfully decoded to bytes; <see langword="false" /> otherwise.</returns>
```